Bugzilla/Token.pm

```diff
@@ -20,7 +20,6 @@ use Bugzilla::User;
 use Date::Format;
 use Date::Parse;
 use File::Basename;
-use Digest::MD5 qw(md5_hex);
```

Comment:
Let's also remove this line in:

robots.cgi
Bugzilla/Template.pm

as they are not even used in those files.

Reply:
I'll handle that later.